feat(php-storybook): Configuring static version, creating script for deploying to gh pages - TWIG-158

planctus opened the pull request:
# PR description

this Pr configures a static build of php-storybook in the php/dist folder and provides the necessary script to deploy that in the gh-pages of our repository.
It was not possible to configure storybook to serve static images like we do in php-storybook, so the resulting files of the build are parsed to remove / from the images path, images are copied in php/dist through an ncp task (to retrieve the files from the symlinks) and simply accessed as a relative path.
The deployed storybook build is available at: https://ec-europa.github.io/ecl-twig

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

papegaill commented:
@planctus I think we need to update the documentation to have the process to use the new commands. I was trying the following `render:php` and then `copy:php` where I get the following errors:
```
$ node_modules/@ecl-twig/php-storybook/resources/get-assets.js
/bin/sh: node_modules/@ecl-twig/php-storybook/resources/get-assets.js: Permission denied
error Command failed with exit code 126.
```
Maybe I forgot a step? It would be best to add the correct procedure to the documentation, what do you think?

Also by running `start:php-ec` I have a bunch of errors relating to `Unexpected closing tag "a"` but it’s definitely another topic.

planctus commented:
Thanks for reviewing this @papegaill , i think that with all the time passed and conflicts fixed we removed some of the scripts needed to use this functionality. The error you had on the getAssets was due to permissions, which are fixed, the errors with prettier were fixed in another pull request already merged, now you see in the package.json the scripts that are meant to deploy on github, i've also updated the documentation to reflect the availability of this functionality.

papegaill commented on docs/getting-started.md:
````diff
@@ -183,7 +183,7 @@ yarn render:js
 
 ```bash
-yarn render
+yarn render-php-js
````
I was not able to run `render-php-js`
```
Script php scripts/render.php handling the render event returned with error code 255
error Command failed with exit code 255.
```
But by running first `start:php-ec` then I was able to `render-php-js`, build dist files and publish to gh-pages. 

Also `error Command "render:setupPhp" not found`, maybe documentation should be updated accordingly.

papegaill replied:
I see in TWIG-180 that the correct cmd is now `setupPhp`
